### PR TITLE
doc(lambda): update nodejs, fix python doc

### PR DESCRIPTION
**Pull request by mat-rumian**
## Purpose of this pull request

update nodejs, python and java documentation for lambda

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/aws-lambda/java.md [hunk 1]
```diff
@@ -9,23 +9,25 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 This document covers how to install and configure OpenTelemetry distributed tracing for AWS Lambda functions based on Java and send the data to Sumo Logic.
 
-## Requirements
+To obtain tracing data from AWS Lambda functions developed in Java you can use [Sumo Logic Distribution for OpenTelemetry Java Lambda](https://github.com/SumoLogic/sumologic-otel-lambda/tree/main/java). It provides auto instrumentation.
```
Comment (JainM6) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/aws-lambda/java.md:12]:
```suggestion
To obtain tracing data from AWS Lambda functions developed in Java, you can use [Sumo Logic Distribution for OpenTelemetry Java Lambda](https://github.com/SumoLogic/sumologic-otel-lambda/tree/main/java). It provides auto instrumentation.  
```

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/aws-lambda/java.md [hunk 2]
```diff
@@ -9,23 +9,25 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
-## Requirements
+To obtain tracing data from AWS Lambda functions developed in Java you can use [Sumo Logic Distribution for OpenTelemetry Java Lambda](https://github.com/SumoLogic/sumologic-otel-lambda/tree/main/java). It provides auto instrumentation.  
 
-You can instrument your AWS Java Lambda function using the [Sumo Logic AWS distro Lambda layer](https://github.com/SumoLogic/opentelemetry-lambda/tree/main/java). By default calls to the Lambda function and AWS Services are instrumented, see the [Manual Instrumentation](#optional-manual-instrumentation) section below if your function is performing some other calls like HTTP requests or database calls.
+**Sumo Logic OTel Java Lambda layer supports:**
+
+* Java8 (Corretto) and Java11 (Corretto) runtimes
+* x86_64 and arm64 architectures
+
+## Sumo Logic Distribution for OpenTelemetry Lambda Layer
```
Comment (JainM6) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/aws-lambda/java.md:19]:
```suggestion
## Sumo Logic Distribution for OpenTelemetry Lambda layer
```

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/aws-lambda/java.md [hunk 3]
```diff
@@ -9,23 +9,25 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+You can instrument your AWS Java Lambda function using the [Sumo Logic Distribution for OpenTelemetry Lambda Layer version 1.24.0](https://github.com/SumoLogic/sumologic-otel-lambda/tree/release-java-v1.24.0/java). By default calls to the Lambda function and AWS Services are instrumented, see the [Manual Instrumentation](#optional-manual-instrumentation) section below if your function is performing some other calls like HTTP requests or database calls.
+
+### Lambda function requirements
 
 You'll need the following:
```
Comment (JainM6) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/aws-lambda/java.md:26]:
```suggestion
```

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/aws-lambda/java.md [hunk 4]
```diff
@@ -9,23 +9,25 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
-You can instrument your AWS Java Lambda function using the [Sumo Logic AWS distro Lambda layer](https://github.com/SumoLogic/opentelemetry-lambda/tree/main/java). By default calls to the Lambda function and AWS Services are instrumented, see the [Manual Instrumentation](#optional-manual-instrumentation) section below if your function is performing some other calls like HTTP requests or database calls.
+**Sumo Logic OTel Java Lambda layer supports:**
+
+* Java8 (Corretto) and Java11 (Corretto) runtimes
+* x86_64 and arm64 architectures
+
+## Sumo Logic Distribution for OpenTelemetry Lambda Layer
+
+You can instrument your AWS Java Lambda function using the [Sumo Logic Distribution for OpenTelemetry Lambda Layer version 1.24.0](https://github.com/SumoLogic/sumologic-otel-lambda/tree/release-java-v1.24.0/java). By default calls to the Lambda function and AWS Services are instrumented, see the [Manual Instrumentation](#optional-manual-instrumentation) section below if your function is performing some other calls like HTTP requests or database calls.
```
Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/aws-lambda/java.md:21]:
```suggestion
You can instrument your AWS Java Lambda function using the [Sumo Logic Distribution for OpenTelemetry Lambda Layer version 1.24.0](https://github.com/SumoLogic/sumologic-otel-lambda/tree/release-java-v1.24.0/java). By default, calls to the Lambda function and AWS Services are instrumented, see the [Manual Instrumentation](#optional-manual-instrumentation) section below if your function is performing some other calls like HTTP requests or database calls.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fix

**Review by kimsauce — APPROVED** [review]

**Pull request merged**